### PR TITLE
fix salt/sodium equivalent issue with unit and modifier, bug #2000

### PR DESCRIPTION
**Pull request by stephanegigandet**

### CODE REVIEW

File: lib/ProductOpener/Food.pm [hunk 1]
```diff
@@ -3861,17 +3863,17 @@ sub fix_salt_equivalent($) {
 		assign_nid_modifier_value_and_unit(
 			$product_ref,
 			'sodium' . $product_type,
-			$product_ref->{nutriments}{'salt' . $product_type} . '_modifier',
-			$product_ref->{nutriments}{'salt' . $product_type} / 2.54,
+			$product_ref->{nutriments}{'salt' . $product_type . '_modifier'},
```
Comment (hangy) [inline lib/ProductOpener/Food.pm]: Ouch 🤦‍♂ sorry.

File: lib/ProductOpener/Food.pm [hunk 2]
```diff
@@ -3861,17 +3863,17 @@ sub fix_salt_equivalent($) {
...
+			$product_ref->{nutriments}{'salt' . $product_type} / 2.5,
```
Comment (hangy) [inline lib/ProductOpener/Food.pm]: https://github.com/openfoodfacts/openfoodfacts-server/blob/3b22df90772f4fc949a19bee65d1113a4f3b4fde/cgi/product_multilingual.pl#L2004

File: lib/ProductOpener/Food.pm [hunk 3]
```diff
@@ -3861,17 +3863,17 @@ sub fix_salt_equivalent($) {
...
 		}
 		elsif ((defined $product_ref->{nutriments}{'sodium' . $product_type}) and ($product_ref->{nutriments}{'sodium' . $product_type} ne '')) {
 			assign_nid_modifier_value_and_unit(
 			$product_ref,
 			'salt' . $product_type,
-			$product_ref->{nutriments}{'sodium' . $product_type} . '_modifier',
-			$product_ref->{nutriments}{'sodium' . $product_type} * 2.54,
-			$product_ref->{nutriments}{'sodium' . $product_type} . '_unit');
+			$product_ref->{nutriments}{'sodium' . $product_type . '_modifier'},
+			$product_ref->{nutriments}{'sodium' . $product_type} * 2.5,
```
Comment (hangy) [inline lib/ProductOpener/Food.pm]: https://github.com/openfoodfacts/openfoodfacts-server/blob/3b22df90772f4fc949a19bee65d1113a4f3b4fde/cgi/product_multilingual.pl#L1999

File: lib/ProductOpener/Food.pm (hunk 1 above)
Comment (stephanegigandet) [reply lib/ProductOpener/Food.pm]: No problem, I didn't see it either.

**Pull request merged**